samples/bot-formatting-cards/csharp/BotFormattingCards/Manifest/manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.15/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.15",
+  "version": "1.0.0",
+  "id": "{{Microsoft-App-Id}}",
+  "packageName": "com.microsoft.teams.differentformattingcards",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "name": {
+    "short": "Different Formatting Cards",
+    "full": "Different Formatting Cards"
+  },
+  "description": {
+    "short": "This sample demonstrates the use of different formatting on cards.",
```

Comment:
This sample shows how to use different formatting on cards.

Reply:
Fixed